File: Itido-BMS/dart/bms_dart/lib/src/models/work_holiday.dart
class WorkHoliday {
  int workContractid;
  String holidayName;
  String holidayCountryCode;

  WorkHoliday({
    this.holidayName,
    this.holidayCountryCode,
  });

  factory WorkHoliday.fromJson(json) {
    if (json == null) return null;
    return WorkHoliday._fromJson(json);
  }

  WorkHoliday._fromJson(json)
      : this.workContractid = json['workContractid'],
        this.holidayName = json['holidayName'],
        this.holidayCountryCode = json['holidayCountryCode'];

  Map<String, dynamic> toMap() => {
        'workContractid': this.workContractid,
        'holidayName': this.holidayName,
        'holidayCountryCode': this.holidayCountryCode,
      };
}

File: Itido-BMS/RentAPI/Rent/Models/TimePlanning/WorkHoliday.cs

namespace Rent.Models.TimePlanning
{

    public class WorkHoliday
    {
        public string HolidayName { get; set; }
        public string HolidayCountryCode { get; set; }
        public int WorkContractID { get; set; }
        public virtual Holiday Holiday { get; set; }
        public virtual WorkContract WorkContract { get; set; }
    }
}